multicluster/charts/linkerd-multicluster-link/templates/service-mirror.yaml

```diff
@@ -9,6 +9,9 @@ metadata:
     mirror.linkerd.io/cluster-name: {{.Values.targetClusterName}}
     {{- with .Values.commonLabels }}{{ toYaml . | trim | nindent 4 }}{{- end }}
 rules:
+- apiGroups: ["coordination.k8s.io"]
+  resources: ["leases"]
+  verbs: ["create", "get", "update", "patch"]
```

Comment:
Leases will be created in the `linkerd-multicluster` namespace, so these privileges can be granted in the Role below instead (I think we made a similar mistake for the policy lease).